webcompat/static/js/lib/bugform.js

```diff
@@ -142,7 +142,7 @@ function BugForm() {
     if (!location.search.search(/open=1/) && !location.search.search(/url=/)) {
       return;
     }
-    var urlParam = location.search.match(/url=(.+)/);
+    var urlParam = location.search.match(/url=([^&]*)/);
```

Comment:
OK, so from the add-on, we `encodeURIComponent` before sticking in the `url` param:

`https://webcompat.com/issues/new?url=http%3A%2F%2Fwww.openstreetmap.org%2Fsearch%3Fquery%3Dparis%23map%3D11%2F48.8591%2F2.2611%26layers%3DC`

That means we wend up with the right thing in the form:
![screen shot 2016-12-28 at 4 03 34 pm](https://cloud.githubusercontent.com/assets/67283/21532585/37345c7a-cd17-11e6-93fa-c00f5343bc62.png)


But if you were to somehow manually type in the URL to the urlbar w/o escaping: `http://localhost:5000/issues/new?url=http://www.openstreetmap.org/search?query=paris#map=11/48.8591/2.2611&layers=C&src=FASTER` we end up with:

`http://www.openstreetmap.org/search?query=paris#map=11/48.8591/2.2611`

So obviously that isn't what we want.

We could either ignore it because it seems like an edgecase, or chop off everything that isn't `&src=foo` (if it's at the end). WDYT, @karlcow?